packages/clerk-js/src/core/auth/AuthCookieService.ts

```diff
@@ -181,18 +181,16 @@ export class AuthCookieService {
   }
 
   private handleGetTokenError(e: any) {
-    //throw if not a clerk api error (aka fapi error) and not a network error
-    if (!isClerkAPIResponseError(e) && !isClerkRuntimeError(e)) {
-      clerkCoreErrorTokenRefreshFailed(e.message || e);
-    }
```

Comment:
will removing this impact the `is4xxError` check below? I think most likely not, but wondering if we want to early return on non clerk API / runtime errors (as this implicitly does today)